### PR TITLE
Master

### PR DESCRIPTION
**Pull request by deemyboy**
Fixed empty subject error from Message.php

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,24 +27,24 @@ backwards compatible.
 
 ```
 "require": {
-  "tedivm/fetch": "0.6.*"
+   "Nadeem Sarwar/fetch": "1.0.*",
```
Comment (sergeyklay) [inline README.md]: Could you please return back this

File: README.md [hunk 2]
```diff
@@ -8,6 +8,7 @@
 Fetch is a library for reading email and attachments, primarily using the POP 
 and IMAP protocols.
 
+# Moved from my personal repo to computerassistance gitlab - Nadeem
```
Comment (sergeyklay) [inline README.md]: ???

File: README.md [hunk 3]
```diff
@@ -64,4 +65,4 @@ by composer.
 
 ## License
 
-Fetch is licensed under the BSD License. See the LICENSE file for details.
+Fetch is licensed under the BSD License. See the LICENSE file for details..
```
Comment (sergeyklay) [inline README.md:66]: Why?

File: src/Fetch/Message.php [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <?php
-
+/* testing edit */
```
Comment (sergeyklay) [inline src/Fetch/Message.php:2]: ???

**Review by sergeyklay — CHANGES_REQUESTED** [review]

**Comment by deemyboy** [comment]
Hello.

I'm new to this so sorry if I've made any mistakes.

What should I do?

**Pull request closed without merging**